apps/remix-ide/src/app/plugins/remix-templates.ts

```diff
@@ -27,6 +27,17 @@ export class TemplatesPlugin extends Plugin {
     const files = await templateWithContent[template](opts, this)
     return files
   }
+
+  async getTemplateReadMeFile(templateName: string) {
+    const files = typeof templateWithContent[templateName] === 'function' ? await templateWithContent[templateName]({}, this) : { 'README.md': `# ${templateName} template` }
+    const readMe = files?.['README.md'] || files?.['README.txt'] || 'No ReadMe file found'
+    return { readMe, type: files['README.md'] ? 'md' : files['README.txt'] ? 'txt' : 'none' }
+  }
+
+  async getBlankTemplateConfigFiles(templateName: string) {
```

Comment:
looks like you are accessing `Blank template`, do you still need `templateName ` as param?